tests/test_host_check.py

```diff
@@ -44,7 +45,7 @@
 }
 
 
-def _checks_to_str(checks) -> str:
+def _checks_to_str(checks: List[host_check.Check]) -> str:  # ignore
```

Comment:
What's the `# ignore` for? This looks like a mypy `# type: ignore` but assume this is doing nothing?